app/models/classroom.rb

```diff
@@ -0,0 +1,18 @@
+class Classroom < ActiveRecord::Base
+  validates :name, presence: true, uniqueness: true, length: { maximum: 255, too_long: "%{count} characters maximum" }
+  has_many :drills, dependent :destroy
+  has_many :drill_types, dependent :destroy
```

Comment:
i think you might mean 
`has_many :drill_types, through: drills`
http://guides.rubyonrails.org/association_basics.html#the-has-many-through-association